modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectFieldResourceTest.java

```diff
@@ -44,16 +41,7 @@ public class ObjectFieldResourceTest extends BaseObjectFieldResourceTestCase {
 	public void setUp() throws Exception {
 		super.setUp();
 
-		String value = "A" + RandomTestUtil.randomString();
-
-		_objectDefinition =
-			_objectDefinitionLocalService.addCustomObjectDefinition(
-				TestPropsValues.getUserId(), 0, false, true,
```

Comment:
Here, the parameters change. It is " TestPropsValues.getUserId(), 0, false, **true**,"

Reply:
Thank you, done!